docs/README.md

```diff
@@ -0,0 +1,59 @@
+# Summary
+
+* [Home](README.md)
+* [Options](./docs/Options.md)
```

Comment:
Wouldn't this nest docs folder twice ? Like, https://github.com/denysdovhan/spaceship-zsh-theme/blob/3.0-docs/docs/docs/Options.md#time-time

Reply:
Nope. `./SUMMARY.md` is link to `./docs/README.md`. GitBook uses `SUMMARY.md` as an entry point. Everything works well here.

Reply:
> Everything works well here.

Then great